Freshly created spring boot project does not compile

**Describe the bug**
A clear and concise description of what the bug is.

When i create a new spring boot project the initial jenkins build fails.

: error: package org.junit.jupiter.api does not exist
import org.junit.jupiter.api.Test;
                            ^
DemoApplicationTests.java:9: error: cannot find symbol
	@Test
	 ^
  symbol:   class Test
  location: class DemoApplicationTests
2 errors

--- a/boilerplates/be-springboot/build.sh
+++ b/boilerplates/be-springboot/build.sh
@@ -1,5 +1,5 @@
 #!/bin/bash
-SPRING_CLI_VERSION=2.0.2.RELEASE
+SPRING_CLI_VERSION=2.2.1.RELEASE
 JENKINS_SLAVE=cd/jenkins-slave-maven:latest
 
 while [[ $# -gt 1 ]]

--- a/boilerplates/be-springboot/templates/build-4.10.gradle
+++ b/boilerplates/be-springboot/templates/build-4.10.gradle
@@ -21,7 +21,7 @@ buildscript {
 
 plugins {
     id 'org.asciidoctor.convert' version '1.5.3'
-    id 'org.springframework.boot' version '2.1.3.RELEASE'
+    id 'org.springframework.boot' version '2.2.1.RELEASE'
     id 'java'
     id 'maven'
     id 'jacoco'

--- a/rundeck-jobs/common/prepare-continuous-integration.yaml
+++ b/rundeck-jobs/common/prepare-continuous-integration.yaml
@@ -54,6 +54,7 @@
         set -eux
         mkdir "/tmp/rundeck_@job.id@_@job.execid@"
         cd "/tmp/rundeck_@job.id@_@job.execid@"
+        echo "globals.bitbucket_sshhost=@globals.bitbucket_sshhost@"
         git clone @globals.bitbucket_sshhost@/opendevstack/ods-project-quickstarters.git
         cd ods-project-quickstarters
         # Here the branch can be changed for testing purposes

